dbcon/joblist/tupleaggregatestep.h

```diff
@@ -156,14 +157,37 @@ class TupleAggregateStep : public JobStep, public TupleDeliveryStep
         {}
         void operator()()
         {
-            utils::setThreadName("TASThrAggr");
+            std::string t{"TASThrAggr"};
+            t.append(std::to_string(fThreadID));
+            utils::setThreadName(t.c_str());
+            //utils::setThreadName("TASThrAggr");
```

Comment:
Plz remove the commented line.

Reply:
Oh, sorry, I missed this. Actually I added thr number to the thread name for debugging (so I was able to see exactly which thread was working on particular bucket). Do you thing it makes sense for production or should I rollback this?